host.py

```diff
@@ -70,24 +75,19 @@ def _is_rsa(self) -> bool:
 
     def login(self) -> paramiko.SSHClient:
         logger.info(f"Logging in into {self._hostname} with {self._key_path}")
-        host = paramiko.SSHClient()
-        host.set_missing_host_key_policy(paramiko.AutoAddPolicy())
```

Comment:
How will removing this line affect behavior. Was this intended to be removed?

Reply:
The lines were found in both derived classes, so I moved it to the base `Login` class. It is now called when the class is initialized.
<https://github.com/bn222/cluster-deployment-automation/pull/162/files#diff-ccff53274e18cb1bf0c3ca0194bf14ee709a8b2a35a74e0ebce33feee69b9c30R46>